cli.js

```diff
@@ -10,18 +10,33 @@ const graphqlHTTP = require('express-graphql')
 const OasGraph = require('./lib/index.js')
 const path = require('path')
 const request = require('request')
+const graphql = require('graphql')
 const fs = require('fs')
 
 const app = express()
 
+const outputFileName = 'schema.graphql';
+
 if (process.argv.length <= 2) {
-  const usage = ' <OAS JSON file path or remote url> [port number]'
-  console.log('Usage: ' + __filename + usage)
+  const usage = 'oasgraph <OAS JSON file path or remote url> [port number] [--save]'
+  console.log('Usage: ' + usage)
   process.exit(-1)
 }
 
 let filePath = process.argv[2]
-let portNumber = process.argv[3] ? process.argv[3] : 3001
+
+// default values if only the url or local file is specified
+let portNumber = 3001
+let saveSchema = false
+
+// examine the last two arguments
+for (var i=3; i<=process.argv.length-1; i++) {
+   if (isNaN(process.argv[i]) && process.argv[i] === '--save') {
+     saveSchema = true
+   } else {
+     portNumber = process.argv[i]
+     } 
```

Comment:
Could you remove this extra space?

Reply:
please check if I understood or specify clearly what extra space you meant?

Reply:
Yes, this is perfect. Thank you for making that change for me!